docker-compose.yml

```diff
@@ -85,6 +85,13 @@ services:
      - redisdata:/data
     networks:
      - harbornet
+# Uncomment selenium section if you want to use selenium (for example for Laravel Dusk ) for your project
+#  selenium:
+#    image: selenium/standalone-chrome:3.11.0-antimony
+#    volumes:
+#      - /dev/shm:/dev/shm
```

Comment:
I think that I found this as recommendation for selenium implementation. Look for example here:
https://sondnm.github.io/blog/2018/09/08/i-just-learnt-about-/dev/shm/ - selenium will be able to use host shared memory ( in cases container consumes too much memory, it can use host shared memory )